library/src/main/java/com/mapzen/valhalla/TransitInfo.kt

```diff
@@ -54,8 +54,12 @@ class TransitInfo {
     return json.optString(KEY_SHORT_NAME)
   }
 
-  fun getColor(): Int {
-    return json.getInt(KEY_COLOR)
+  fun getColor(): Int? {
+    if (json.has(KEY_COLOR)) {
```

Comment:
I didn't actually observe a place where color wasn't available in the wild (I did run into this when writing the tests) so I figured just in case its probably good practice to validate the input.